pkg/signature/kms/cliplugin/common/options.go

```diff
@@ -27,6 +27,12 @@ type SignOptions struct {
 	MessageOptions MessageOptions `json:"messageOptions"`
 }
 
+// VerifyOptions contains the values for signature.VerifyOption.
+type VerifyOptions struct {
+	RPCOptions     RPCOptions     `json:"rpcOptions"`
+	MessageOptions MessageOptions `json:"messageOptions"`
```

Comment:
To confirm, if I want to verify an object without passing its digest, this is passed over stdin?